bin/deploy-dockerhub.sh

```diff
@@ -26,13 +26,27 @@ echo "$DOCKER_PASS" | docker login -u="$DOCKER_USER" --password-stdin
 
 docker images
 
+SERVER_CONTAINER="remotesettings/server"
+TEST_CONTAINER="remotesettings/tests"
+SERVER_DOCKER_REPO="mozilla/remote-settings"
+INTEGRATION_TEST_DOCKER_REPO="mozilla/remote-settings-integration-tests"
+
 # docker tag and push git branch to dockerhub
 if [ -n "$1" ]; then
     TAG="$1"
-    echo "Tag and push ${DOCKERHUB_REPO}:${TAG} to Dockerhub"
-    docker tag remotesettings:server "$DOCKERHUB_REPO:$TAG" ||
-        (echo "Couldn't tag remote-settings as $DOCKERHUB_REPO:$TAG" && false)
-    retry 3 docker push "$DOCKERHUB_REPO:$TAG" ||
-        (echo "Couldn't push $DOCKERHUB_REPO:$TAG" && false)
+    echo "Tag and push server and integration test containers to Dockerhub"
+    echo "${SERVER_DOCKER_REPO}:${TAG}"
+    echo "${INTEGRATION_TEST_DOCKER_REPO}:${TAG}"
+
+    docker tag $SERVER_CONTAINER "$SERVER_DOCKER_REPO:$TAG" ||
+        (echo "Couldn't tag $SERVER_CONTAINER as $SERVER_DOCKER_REPO:$TAG" && false)
+    retry 3 docker push "$SERVER_DOCKER_REPO:$TAG" ||
+        (echo "Couldn't push $SERVER_DOCKER_REPO:$TAG" && false)
+
+    docker tag $TEST_CONTAINER "$INTEGRATION_TEST_DOCKER_REPO:$TAG" ||
+        (echo "Couldn't tag $TEST_CONTAINER as $INTEGRATION_TEST_DOCKER_REPO:$TAG" && false)
+    retry 3 docker push "$INTEGRATION_TEST_DOCKER_REPO:$TAG" ||
+        (echo "Couldn't push $INTEGRATION_TEST_DOCKER_REPO:$TAG" && false)
+
```

Comment:
I have the feeling that we could have a bash function instead of duplicating everything

Reply:
Good call. Added here d3554ffc55db98f6b785394c7af6e6d4423ca5cf.